src/PartEditor/PartGraphSelector.ts

```diff
@@ -90,8 +91,12 @@ export class PartGraphSelPanel extends CircleGraphCtrl implements CircleGraphEve
 
   public static createOrShow(
       extensionUri: vscode.Uri, args: PartGraphCmdOpenArgs, handler: PartGraphEvent|undefined) {
-    const column =
-        vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
+    let column = args.viewColumn ? args.viewColumn : undefined;
+    if (column) {
+      if (column >= vscode.ViewColumn.One) {
+        column = column + 1;
+      }
+    }
```

Comment:
(optional) I'm not sure but doesn't it have a same meaning?
```suggestion
    let column = args.viewColumn
    if (column) {
      if (column >= vscode.ViewColumn.One) {
        column = column + 1;
      }
    }
```